ApiFirst/ApiFirst/Controllers/CostumersController.cs

```diff
@@ -0,0 +1,89 @@
+﻿using DomainModels.Models;
+using DomainServices.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+
+namespace WebApi.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class CustomersController : ControllerBase
+    {
+        private readonly ICustomerRepository _service;
+
+        public CustomersController(ICustomerRepository repo)
+        {
+            _service = repo ?? throw new ArgumentNullException(nameof(repo));
+        }
```

Comment:
É service ou repository?
```suggestion
        private readonly ICustomerRepository _customerRepository;

        public CustomersController(ICustomerRepository customerRepository)
        {
            _customerRepository = customerRepository?? throw new ArgumentNullException(nameof(customerRepository));
        }
```